packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue

```diff
@@ -68,78 +76,97 @@
                     color-scheme="positive" />
             </div>
 
+            <template v-if="isLoading">
+                <restaurant-content-skeleton
+                    :width-percentage="90"
+                    :class="[
+                        $style['c-restaurantCard-content--left-col'],
+                        $style['c-restaurantCard--hideOnMobile']]" />
+                <restaurant-content-skeleton
+                    :width-percentage="80"
+                    :class="[
+                        $style['c-restaurantCard-content--left-col'],
+                        $style['c-restaurantCard--hideOnMobile']]" />
+                <restaurant-content-skeleton
+                    :width-percentage="40"
+                    :class="[
+                        $style['c-restaurantCard--hideOnMobile']]" />
+                <restaurant-content-skeleton :width-percentage="90" />
+            </template>
+
+            <template v-else>
+                <!-- Cuisines -->
+                <component
+                    :is="errorBoundary"
+                    v-if="hasCuisines"
+                    :tier="3">
+                    <restaurant-cuisines
+                        :class="[$style['c-restaurantCard-content--left-col'], $style['c-restaurantCard-cuisines']]"
```

Comment:
nit.--> This is formatted differently to the class declarations above. Might want to make the same

Reply:
good spot - updating now